src/audio/AudioAnalyser.js

```diff
@@ -20,6 +20,22 @@ Object.assign( THREE.AudioAnalyser.prototype, {
 		this.analyser.getByteFrequencyData( this.data );
 		return this.data;
 
+	},
+
+	getAverage: function() {
```

Comment:
Maybe I did not update the cache. I reload and show:
```
Uncaught IndexSizeError: Failed to set the 'fftSize' property on 'AnalyserNode': The FFT size provided (2) is outside the range [32, 32768].
```
In console message. The the minimum is 32.